packages/expo-clipboard/src/Clipboard.ts

```diff
@@ -170,17 +175,36 @@ export async function hasImageAsync(): Promise<boolean> {
  * is a no-op on Web.
  *
  * @param listener Callback to execute when listener is triggered. The callback is provided a
- * single argument that is an object with a `content` key.
+ * single argument that is an object containing information about clipboard contents.
  *
  * @example
  * ```typescript
- * addClipboardListener(({ content }: ClipboardEvent) => {
- *   alert('Copy pasta! Here's the string that was copied: ' + content);
+ * Clipboard.addClipboardListener(({ contentTypes }: ClipboardEvent) => {
+ *   if (contentTypes.includes(Clipboard.ContentType.PLAIN_TEXT)) {
+ *     Clipboard.getStringAsync().then(content => {
+ *       alert('Copy pasta! Here's the string that was copied: ' + content)
+ *     });
+ *   } else if (contentTypes.includes(Clipboard.ContentType.IMAGE)) {
+ *     alert('Yay! Clipboard contains an image');
+ *   }
  * });
  * ```
  */
 export function addClipboardListener(listener: (event: ClipboardEvent) => void): Subscription {
-  return emitter.addListener<ClipboardEvent>(onClipboardEventName, listener);
+  // TODO: Get rid of this wrapper once we remove deprecated `content` property
```

Comment:
Let's add more info when we can remove this. I suggest to keep it at least for next two cycles

```suggestion
  // TODO: Get rid of this wrapper once we remove deprecated `content` property (not before SDK47)
```

Reply:
```suggestion
  // TODO(SDK47+): Get rid of this wrapper once we remove deprecated `content` property
```

Reply:
This is not always correct — for example Xcode's standard needs the colon to be exactly after the word, like `// TODO:`, `// MARK:` or `// FIXME:` and I think it would be great to have just one rule for that.